### DISCUSSION TITLE
轻松水到60级

### DISCUSSION THREAD

**Discussion by dengyunzhe** [share121/inter-knot#1356] [其他]
有图为证
![1](https://pic.imgdb.cn/item/66b85eb8d9c307b7e91fe5fd.png)

**Comment by su2569** [comment]
`<div class="level-num" data-v-af4c8e32="">1</div>`
改成
`<div class="level-num" data-v-af4c8e32="">60</div>`

**Comment by su2569** [comment]
然后就
![github源](https://github.com/user-attachments/assets/08cfae77-9124-41dc-b7a1-51bf3fe7d23b)
![国内源（可能被改）](https://f.cdn-static.cn/81470_17233610517983.png)

**Comment by EhuanyingK** [comment]
666，这个入开桂了

**Comment by su2569** [comment]
经验是把
`<div class="bar" style="width:68.5%;" data-v-af4c8e32=""><span class="curExp" data-v-af4c8e32="">6850</span> / <span class="totalExp" data-v-af4c8e32="">10000</span></div>`
改成
`<div class="bar" style="width:100%;" data-v-af4c8e32=""><span class="curExp" data-v-af4c8e32="">10000</span> / <span class="totalExp" data-v-af4c8e32="">10000</span></div>`

**Comment by 0721Lover** [comment]
+3

**Comment by muxueano** [comment]
就你偷窥我是吧
![Uploading 2K.9.png…]()
